src/views/bibliography_element.ts

```diff
@@ -240,10 +240,14 @@ export class BibliographyElementBlockView<
     dataTracked: TrackedAttrs
   ) => {
     if (dataTracked.status !== CHANGE_STATUS.rejected) {
-      element.addEventListener('click', () => {
-        this.view.dispatch(
-          this.view.state.tr.setMeta(SET_SUGGESTION_ID, dataTracked.id)
-        )
+      element.addEventListener('click', (e) => {
+        console.log(e)
+        console.log('CLICK')
+        setTimeout(() => {
```

Comment:
are you sure we need `setTimeout`? it will work without it

Reply:
Yes, we don't need that, it was some testing that I did during the implementation. The problem is that there was another listener on that button that was supposed to reveal the TC modal but it didn't work because dispatch would recreate that html and it worked only with the delay. But since we are rendering the TC modal now only for the inspector it's irrelevant.